### ISSUE TITLE
L2_Normalization

### ISSUE THREAD

**Issue by YuanEZhou** [asdf0982/vqa-mfb.pytorch#4]
Hi @asdf0982,  thanks for your helpful code.  I have a question. Specifically, I think it's more reasonable to replace **the line 78-80 of mfb_coatt_glove.py** with  **iatt_iq_l2=F.normalize(iatt_iq_sqrt,dim=1)** to do the L2_normalization operation.  Do you think so?

**Comment by asdf0982** [comment]
@YuanEZhou emmmmm...I think different feature boxes of an image have different importance. If you normalize them separately, the sum of each feature box will be same, it will lead to a decline in accuracy, I guess.
